src/wp-includes/post.php

```diff
@@ -7893,6 +7893,27 @@ function _prime_post_caches( $ids, $update_term_cache = true, $update_meta_cache
 
 		update_post_caches( $fresh_posts, 'any', $update_term_cache, $update_meta_cache );
```

Comment:
```suggestion
		update_post_caches( $fresh_posts, 'any', false, false );
```

Doesn't this need to be this now?

Reply:
> Doesn't this need to be this now?

No. Post IDs that are passed to `update_post_caches()` are excluded from the fallback checks due to the `array_diff()` below.

Reply:
We could set it to `false` but have to remove the `array_diff()` call. And then again we could just use `update_post_cache()`. There are a way too many similar functions to be honest…

But I think this way it's a bit cleaner.